### PR TITLE
Javascript interface

### PR DESCRIPTION
**Pull request by taiar**
This PR provides a JavaScript interface to call trackers in a JavaScript based interface.

### CODE REVIEW

File: app/assets/javascripts/action_tracker.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  }
+
+  function start(list) {
+    storage.queue(list);
+  }
+
+  function process() {
+    while(typeof storage.getFirst() !== 'undefined') {
+      var tracker = new Tracker(storage.getFirst());
+      tracker.send();
```
Comment (antoniosb) [inline app/assets/javascripts/action_tracker.js:22]: This send doesn't really ensure that the item popped from the storage, right? Why you still need to get it then dequeue it?

Comment (taiar) [inline app/assets/javascripts/action_tracker.js:22]: Refactored.

**Comment by antoniosb** [comment]
:+1:

**Comment by joaofraga** [comment]
:+1:

**Comment by joaofraga** [comment]
Great job here! Congratulations... 👏 👏 👏 👏 👏

**Comment by taiar** [comment]
Thank you @joaofraga!

**Pull request merged**